learnyounode/exercises/05-filtering-function.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+let fs = require('fs');
+
+let filter = function() {
+  if (error) {
+    throw console.error(error);
+  }
+
+  fs.readdir(process.argv[2], function(err, list) {
+    list.forEach(function(file) {
+      if (path.extname(file) === '.' + process.argv[3]) {
```

Comment:
I'm not sure if `process.argv` is available from the interior module. In any case, it's not great separation of concerns to put it all together. My recommendation would have been to keep reading `process.argv` and printing the output separate from the filtering operation.